bin/notification.js

```diff
@@ -0,0 +1,79 @@
+const async = require('async');
+const schedule = require('node-schedule');
+
+const werelogs = require('werelogs');
+
+const config = require('../conf/Config');
+const zkConfig = config.zookeeper;
+const kafkaConfig = config.kafka;
+const extConfigs = config.extensions;
+const qpConfig = config.queuePopulator;
+const httpsConfig = config.internalHttps;
+const mConfig = config.metrics;
+const rConfig = config.redis;
```

Comment:
Alternate syntax, but I'm fine with either.
```suggestion
const {
    zookeeper: zkConfig,
    kafka: kafkaConfig,
    extensions: extConfigs,
    queuePopulator: qpConfig,
    internalHttps: httpsConfig,
    metrics: mConfig,
    redis: rConfig,
} = config;
```